### ISSUE TITLE
Latest version of Scratch 3 breaks mlforkids extensions

### ISSUE THREAD

**Issue by dalelane** [IBM/taxinomitis#205]
ML blocks aren't returning any results. 

Not sure of the details of this yet. It was reported by a user, but I have been able to reproduce. 

Another unusual symptom was that the ML blocks were the default green, not the custom dark-blue-black colour they should be. Don't know if that's a related symptom or a red herring. 

**_For now, I've reverted to a previous Scratch 3 build so this shouldn't be affecting anyone any more._**

**Comment by dalelane** [comment]
If you still see the machine learning blocks coloured in green in Scratch 3, like the photo below, please make sure you refresh the page. You might need to clear your cache if that doesn't work. 

Green blocks seem to be a symptom of the broken version. 

**Saving your project to a file before refreshing will work - so you don't have to lose any work to do this.**

![image](https://user-images.githubusercontent.com/1444788/61127950-9acb8180-a4a8-11e9-9922-b9c3ca79238c.png)

**Comment by quell27** [comment]
Still no luck with image recognition in scratch, will try again tomorrow.

**Comment by dalelane** [comment]
@quell27 Is that in Scratch 3? Did you confirm that the blocks were the right colour (not green)? 

If so, then this is a new and/or different problem - so would not be any different tomorrow. If you would be willing to send me a copy of your project file, I would be happy to investigate.

**Comment by MustafAdlin** [comment]
It works now for me. Try to clear cookies and cached files. Thanks

**Comment by dalelane** [comment]
I've identified the change in Scratch 3 that broke the ML blocks : https://github.com/LLK/scratch-vm/commit/a115f6dde50cc6ae5a01582b880da52efdc40232

This first made it into a build a couple of days ago. It was fortunately not hit by all schools, as browsers cache Scratch aggressively (for one week) so most schools will have still been using the old version. It also didn't affect ScratchX/Scratch 2 users. 

This will have predominantly affected new users trying Scratch 3. 

I've created some test projects so I can do a better job of testing new versions of Scratch on future before they can affect anyone.
